teradatalabs/mapr52-base/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM teradatalabs/centos6-java8-oracle
+MAINTAINER Teradata Docker Team <docker@teradata.com>
+
+
+#Add repo for mapr
+ADD files/maprtech.repo /etc/yum.repos.d/maprtech.repo
+RUN yum update -y
+
+#install utility softwares
+RUN yum install iputils vim openssh-server openssh-clients sudo -y
+
+#configure sshh
+RUN chkconfig sshd on
+RUN service sshd start
+
+#get maprgpg key
+RUN rpm --import http://package.mapr.com/releases/pub/maprgpg.key
+
+#add user and password
+RUN adduser mapr
```

Comment:
Adding user and creating a home directory can be merged into one `-d`. Also looks like you can merge many of these commands. Try to combine them as far as possible